packages/keychain/tests/wallet.test.ts

```diff
@@ -257,4 +267,5 @@ test('restoreIdentities', async () => {
   expect(wallet.identities[0].defaultUsername).toEqual('hankstoever.id');
   expect(wallet.identities[1].defaultUsername).toEqual('hankstoever2.id');
   expect(wallet.identities[2].defaultUsername).toEqual('hankstoever3.id');
+  done();
```

Comment:
Was there an issue here that necessitated the explicit `done` call?

Reply:
I can remove this now, added it while trying to fix the tests, [the setup import was missing](https://github.com/blockstack/ux/pull/412/commits/66f8a3fbb1dd27c8b42ad75693d7469631a7190a#diff-e972808428b3c57d74c9980ccab33f8aR1)